_posts/2021-02-16-cockpit-238.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 238
+author: kkoukiou
+date: 2021-02-17
+tags: cockpit linux
+slug: cockpit-238
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 238.
+
+### Updates: List outdated software that needs a restart
+
+Previously, we required a system reboot after every update. We now use [Tracer](http://tracer-package.com/)
+to display the list of services and applications that are outdated and need to be restarted or require a system reboot.
+
+If there are not such services not system reboot will be suggested.
+
+![Update no restart](/images/update-no-restart.png)
+
+If there are such services, the user will be asked to either to restart these or schedule a system reboot.
+
+![Update with restart](/images/update-with-restart.png)
+
+### Web server: Preserve permissions of administrator-provided certificates
+
+Cockpit's web server supports and encourages [using your own TLS certificate/key](https://cockpit-project.org/guide/latest/https.html#https-certificates) in `/etc/cockpit/ws-certs.d/`.
+Previously, starting cockpit would adjust the file permissions and SELinux context of administrator provided
+certificates.  But commonly such files are shared with other services, in which case the permission changes could render
+the file inaccessible for these other services.
+
+Now cockpit only adjusts the permissions of certificate files which it creates and manages itself, in particular
+`0-self-signed.cert` and `10-ipa.cert`. If you provide your own certificate, you must make sure yourself that these are
+readable by the `cockpit-ws` user or group.
+
+### System: Performance page shows up to 16 cores
```

Comment:
I think we should write 'shows the busiest CPU cores'

Reply:
```suggestion
### System: Performance page shows busiest CPU cores
```

OK :+1: